requirements.txt

```diff
@@ -1,18 +1,62 @@
-bandit==1.7.1
+alembic==1.8.0
+attrs==21.4.0
+bandit==1.7.4
+black==22.3.0
+certifi==2022.6.15
+cfgv==3.3.1
+chardet==4.0.0
+charset-normalizer==2.0.12
+click==8.1.3
+clickclick==20.10.2
+colorama==0.4.5
+commonmark==0.9.1
+connexion==2.13.1
+distlib==0.3.4
+dparse==0.5.1
+filelock==3.7.1
+Flask==2.1.2
+Flask-Migrate==3.1.0
+Flask-SQLAlchemy==2.5.1
+fsd-tech @ git+https://github.com/communitiesuk/funding-service-design-utils.git@c35a1362832617ca182a7d71683726a37b0fd6f7
+gitdb==4.0.9
+GitPython==3.1.27
+greenlet==1.1.2
+identify==2.5.1
+idna==3.3
+inflection==0.5.1
+itsdangerous==2.1.2
+Jinja2==3.1.2
+jsonschema==4.6.0
+Mako==1.2.0
+MarkupSafe==2.1.1
+mypy-extensions==0.4.3
+nodeenv==1.6.0
+openapi-schema-validator==0.2.3
+openapi-spec-validator==0.4.0
+packaging==21.3
+pathspec==0.9.0
+pbr==5.9.0
+platformdirs==2.5.2
+prance==0.21.8.0
+pre-commit==2.19.0
+Pygments==2.12.0
+pyparsing==3.0.9
+pyrsistent==0.18.1
+python-dotenv==0.20.0
+PyYAML==6.0
+requests==2.28.0
+rich==12.4.4
+ruamel.yaml==0.17.21
+ruamel.yaml.clib==0.2.6
 safety==1.10.3
-werkzeug==2.0.3
```

Comment:
We have similar problems in all the repos with random usage of `pip freeze`
The approach we took on digital marketplace was using `pip-compile` with a `requirements.in` file including the top level dependencies https://github.com/jazzband/pip-tools#example-usage-for-pip-compile I can spin off a card to adopt this approach if people like the sound of it?

Reply:
I'm happy with this 👍

Reply:
sounds good

Reply:
This still seems to be the full pip freeze - are we agreed it would be worth sliming this down (with whatever package we use)? If we don't do it now in this ticket, it might be worth making a ticket to add pip-compile to repos.

Reply:
have pip compiled this now 👍 happy for us to add a ticket on the board to get this consistent across the repos 💯